crates/amaru-ledger/tests/data/transactions/preprod/7a098c13f3fb0119bc1ea6a418af3b9b8fef18bb65147872bf5037d28dda7b7b/unknown-reference-input/expected.traces

```diff
@@ -1,6 +1 @@
-[ {"name":"require_witness_span","credential": {"hash":"5309fa786856c1262d095b89adf64fe8a5255ad19142c9c537359e41","type":"verification_key"}}
-, {"name":"require_witness_span","credential": {"hash":"4e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e749","type":"script"}}
-, {"name":"require_witness_span","credential": {"hash":"4e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e749","type":"script"}}
-, {"name":"require_witness_span","credential": {"hash":"4e3b75eb68829ca9e63b541f3645f78afe8d119932a7a2528171e749","type":"script"}}
-, {"name":"require_witness_span","credential": {"hash":"5309fa786856c1262d095b89adf64fe8a5255ad19142c9c537359e41","type":"verification_key"}}
-]
+[]
```

Comment:
🤔 ?

Reply:
The ordering of the input validation changed, so now traces aren't happening when there is an err found in the reference inputs